snuba/datasets/transactions_processor.py

```diff
@@ -155,6 +155,31 @@ def process_message(
                     extra={"measurements": measurements},
                     exc_info=True,
                 )
+
+        breakdowns = data.get("breakdowns")
+        if breakdowns is not None:
+            span_op_breakdowns = breakdowns.get("span_ops")
+            if span_op_breakdowns is not None:
+                try:
+                    (
+                        processed["span_op_breakdowns.key"],
+                        processed["span_op_breakdowns.value"],
+                    ) = extract_nested(
+                        span_op_breakdowns,
+                        lambda value: float(value["value"])
+                        if (
+                            value is not None
+                            and isinstance(value.get("value"), numbers.Number)
+                        )
+                        else None,
+                    )
+                except Exception:
+                    logger.error(
```

Comment:
Please use a metric here instead of an error. If this ends up needing to be debugged we can add more logging. We can change this back to an error later.

Reply:
I've removed the try-catch from here. I'll do the same for the measurements.